src/android/src/com/chariotsolutions/nfc/plugin/NfcPlugin.java

```diff
@@ -155,18 +145,6 @@ public boolean execute(String action, JSONArray data, CallbackContext callbackCo
         } else if (action.equalsIgnoreCase(ERASE_TAG)) {
             eraseTag(callbackContext);
 
-        } else if (action.equalsIgnoreCase(SHARE_TAG)) {
-            shareTag(data, callbackContext);
-
-        } else if (action.equalsIgnoreCase(UNSHARE_TAG)) {
-            unshareTag(callbackContext);
```

Comment:
OK to remove `handover`/`stopHandover` because directly related to Beam, as visible in the code here.
But why removing `share`/`unshare`? It seems to be related to "NdefPush"/"Push" ; is that part of Android Beam too?

Reply:
The removal of shareTag and unshareTag actions is due to their dependence on the NDEF push feature, which is primarily facilitated through Android Beam. Since Android Beam and related functionalities have been deprecated and removed in newer Android APIs, maintaining these actions would lead to future compatibility issues and potential security risks. This ensures our plugin stays updated with supported Android NFC capabilities.

Reply:
Android Beam [https://developer.android.com/develop/connectivity/nfc/nfc#p2p](url) a feature that utilized NFC for file sharing, is no longer supporte. Although it still exists in the SDK, it is not functional; the code does not compile and links to the documentation are non-existent. 
Development guides once referred to "Beam file," [https://developer.android.com/training/beam-files](url) but this documentation has been removed and redirects to 'file sharing'. Specific links to Android Beam [https://developer.android.com/reference/android/nfc/NfcAdapter](url) as well as any dedicated documentation and its specific methods are no longer available.